src/Modules/Data/Commands.Data.Test/GetPowerBITableTests.cs

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.PowerBI.Commands.Common.Test;
+using Microsoft.PowerBI.Commands.Profile.Test;
+using Microsoft.PowerBI.Common.Abstractions;
+using Microsoft.PowerBI.Common.Api;
+using Microsoft.PowerBI.Common.Api.Datasets;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using System.Linq;
+
+namespace Microsoft.PowerBI.Commands.Data.Test
+{
+    [TestClass]
+    public class GetPowerBITableTests
+    {
+        public static CmdletInfo GetPowerBITableCmdletInfo { get; } = new CmdletInfo($"{GetPowerBITable.CmdletVerb}-{GetPowerBITable.CmdletName}", typeof(GetPowerBITable));
+        public static CmdletInfo GetPowerBIDatasetCmdletInfo { get; } = new CmdletInfo($"{GetPowerBIDataset.CmdletVerb}-{GetPowerBIDataset.CmdletName}", typeof(GetPowerBIDataset));
```

Comment:
I think these should be private and you might be able to convert them to use the => Operator. See AddPowerBIWorkspaceUserTests.cs as an example.

Reply:
I updated. I originally copied from GetPowerBIDatasetTest.cs so we may need to update that one, too.

Reply:
That's right. I overlooked this when I reviewed that file previously.